reproducible_build.sh

```diff
@@ -3,27 +3,51 @@
 set -euo pipefail
 
 function HELP {
-    echo "Build a lbrycrd"
+    echo "Build lbrycrd"
     echo "-----"
     echo "When run without any arguments, this script expects the current directory"
     echo "to be the lbrycrd repo and it builds what is in that directory"
     echo
+    echo "This is a long build process so it can be split into two parts"
+    echo "Specify the -d flag to build only the dependencies"
+    echo "and the -l flag to build only lbrycrd. This will fail"
+    echo "if the dependencies weren't built earlier"
+    echo
     echo "Optional arguments:"
     echo
     echo "-c: clone a fresh copy of the repo"
-    echo "-h: show help"
+    echo "-r: remove intermediate files."
+    echo "-l: build only lbrycrd"
+    echo "-d: build only the dependencies"
+    echo "-o: timeout build after 45 minutes"
```

Comment:
used on travis.  If the build doesn't stop on its own, the cache won't get populated and we'll never get this thing built.